app/components/learnergroup-user-manager.js

```diff
@@ -16,6 +16,15 @@ export default class LearnergroupCohortUserManagerComponent extends Component {
     return this.args.sortBy.search(/desc/) === -1;
   }
 
+  get selectableUsers() {
+    if (this.currentUser.isRoot) {
+      return this.args.users.mapBy("content");
+    }
+    return this.args.users.filter((user) => {
+      return user.get("enabled");
```

Comment:
This `get` isn't necessary, can use be `user.enabled`.

Reply:
nyet. `user` is an ember proxy object, so we still have to use the getter.
```
Error: Assertion Failed: You attempted to access the `enabled` property (of <(unknown):ember28001>).
Since Ember 3.1, this is usually fine as you no longer need to use `.get()`
to access computed properties. However, in this case, the object in question
is a special kind of Ember object (a proxy). Therefore, it is still necessary
to use `.get('enabled')` in this case.
```

Reply:
Ah ok. Sorry!